tests/playwright/shiny/bugs/2061-css-js-inclusion/www/customjs.js

```diff
@@ -0,0 +1 @@
+console.log("heyo")
```

Comment:
Have this append a message in the UI, like 'Passed'. Then have a test to assert the message has been added.